client/src/components/LoginForm.jsx

```diff
@@ -1,32 +1,15 @@
 import React, { useState, useContext } from 'react';
-import { Grid, Typography, FormControl, InputLabel, OutlinedInput, InputAdornment, IconButton, Button, Container } from '@mui/material';
-import Visibility from '@mui/icons-material/Visibility';
-import VisibilityOff from '@mui/icons-material/VisibilityOff';
-import { makeStyles } from '@mui/styles';
+import { BiHide, BiShow } from "react-icons/bi";
```

Comment:
You can keep this here because the icons look virtually the same, but we've been using `lucide-react` for the icons